fdbclient/StorageServerInterface.cpp

```diff
@@ -296,6 +296,27 @@ void TSS_traceMismatch(TraceEvent& event,
 	ASSERT(false);
 }
 
+// split range
+template <>
+bool TSS_doCompare(const RangeFeedReply& src, const RangeFeedReply& tss) {
+	ASSERT(false);
```

Comment:
It seems like we probably want to actually do a TSS comparison here if there is a TSS, for load duplication and because it would find mismatches in the range feed? (this can definitely be handed later though too)